bluesky/examples.py

```diff
@@ -21,6 +21,7 @@ def describe(self):
 
 class Base:
     def __init__(self, name, fields):
+        self._name = name
```

Comment:
@danielballan Something was expecting these objects to have a `_name` attribute.  I did not dig at all to see if the `.name` attribute is still used.  i guess the question here is should this be `_name` or `name`?